plugins/obs-filters/obs-filters.c

```diff
@@ -5,42 +5,67 @@ OBS_DECLARE_MODULE()
 
 OBS_MODULE_USE_DEFAULT_LOCALE("obs-filters", "en-US")
 
-extern struct obs_source_info mask_filter;
+extern struct obs_source_info async_delay_filter;
+extern struct obs_source_info chroma_key_filter;
+extern struct obs_source_info color_filter;
+extern struct obs_source_info color_grade_filter;
+extern struct obs_source_info color_key_filter;
+extern struct obs_source_info compressor_filter;
 extern struct obs_source_info crop_filter;
+extern struct obs_source_info crosshatching_filter;
+extern struct obs_source_info cross_stitching_filter;
+extern struct obs_source_info droste_filter;
 extern struct obs_source_info gain_filter;
-extern struct obs_source_info color_filter;
-extern struct obs_source_info scale_filter;
-extern struct obs_source_info scroll_filter;
 extern struct obs_source_info gpu_delay_filter;
-extern struct obs_source_info color_key_filter;
-extern struct obs_source_info color_grade_filter;
-extern struct obs_source_info sharpness_filter;
-extern struct obs_source_info chroma_key_filter;
-extern struct obs_source_info async_delay_filter;
+extern struct obs_source_info like_a_dream_filter;
+extern struct obs_source_info luma_key_filter;
+extern struct obs_source_info mask_filter;
+extern struct obs_source_info night_vision_filter;
+extern struct obs_source_info noise_gate_filter;
+
```

Comment:
I personally don't see the need for blank lines surrounding the `#if` preprocessor directives here and below.  I'm not aware of a specifically preferred style for this project, but previous versions of this file don't seem to have them.  See the [plugins/obs-filters/obs-filters.c@20.0.1](https://github.com/jp9000/obs-studio/blob/20.0.1/plugins/obs-filters/obs-filters.c) for comparison.